adhocteam/Head-Start-TTADP
# [NO-TICKET]: Move <Editor /> outside of <p> elements since divs should not be inside paragraph elements

## Description of change


Just moving some `Editor` components outside of some `<p>` elements.

## How to test

Look at code on master, see warnings about nesting divs inside paragraphs. Check out this branch and those warnings disappear.

## Issue(s)

* Follow up to https://ocio-jira.acf.hhs.gov/browse/TTAHUB-26


## Checklists

### Every PR



### Production Deploy


### After merge/deploy
